src/overlays/actors/ovl_kaleido_scope/z_kaleido_scope_PAL.c

```diff
@@ -167,21 +167,149 @@ static u16 D_8082ABEC[] = {
     PAUSE_MAP, PAUSE_EQUIP, PAUSE_QUEST, PAUSE_ITEM, PAUSE_EQUIP, PAUSE_MAP, PAUSE_ITEM, PAUSE_QUEST,
 };
 
+// Controls if the item slot specifided is able to be equipped and the color of the name card
 u8 gSlotAgeReqs[] = {
-    1, 9, 9, 0, 0, 9, 1, 9, 9, 0, 0, 9, 1, 9, 1, 0, 0, 9, 9, 9, 9, 9, 0, 1,
+    AGE_REQ_CHILD, // Deku stick
+    AGE_REQ_NONE,  // Deku Nuts
+    AGE_REQ_NONE,  // Bombs,
+    AGE_REQ_ADULT, // Bow
+    AGE_REQ_ADULT, // Fire arrows
+    AGE_REQ_NONE,  // Dins fire,
+    AGE_REQ_CHILD, // Slingshot
+    AGE_REQ_NONE,  // Ocarina,
+    AGE_REQ_NONE,  // Bombchus
+    AGE_REQ_ADULT, // Hookshot
+    AGE_REQ_ADULT, // Ice arrows
+    AGE_REQ_NONE,  // Farore's Wind
+    AGE_REQ_CHILD, // Boomerang
+    AGE_REQ_NONE,  // Lens of truth
+    AGE_REQ_CHILD, // Bean
+    AGE_REQ_ADULT, // Megaton Hammer
+    AGE_REQ_ADULT, // Light arrows
+    AGE_REQ_NONE,  // Nayrus love
+    AGE_REQ_NONE,  // Bottle 1
+    AGE_REQ_NONE,  // Bottle 2,
+    AGE_REQ_NONE,  // Bottle 3,
+    AGE_REQ_NONE,  // Bottle 4,
+    AGE_REQ_ADULT, // Adult trade
+    AGE_REQ_CHILD  // Child trade
 };
 
 u8 gEquipAgeReqs[][4] = {
-    { 0, 1, 0, 0 },
-    { 9, 1, 9, 0 },
-    { 0, 9, 0, 0 },
-    { 9, 9, 0, 0 },
-};
-
-u8 gItemAgeReqs[] = {
-    1, 9, 9, 0, 0, 9, 1, 9, 9, 9, 0, 0, 0, 9, 1, 9, 1, 0, 0, 9, 9, 9, 9, 9, 9, 9, 9, 9, 9,
-    9, 9, 9, 9, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
-    0, 1, 0, 0, 1, 9, 0, 9, 0, 0, 9, 0, 0, 1, 1, 1, 0, 0, 0, 9, 9, 9, 1, 0, 0, 9, 9, 0,
+    {
+        AGE_REQ_ADULT, // Quiver/Bullet bag (Unused)
+        AGE_REQ_CHILD, // Kokiri sword
+        AGE_REQ_ADULT, // Master Sword
+        AGE_REQ_ADULT  // Big Goron's Sword
+    },
+    {
+        AGE_REQ_NONE,  // Bomb bag
+        AGE_REQ_CHILD, // Deku shield
+        AGE_REQ_NONE,  // Hylian shield
+        AGE_REQ_ADULT  // Mirror shield
+    },
+    {
+        AGE_REQ_ADULT, // Strength (Unused)
+        AGE_REQ_NONE,  // Kokiri tunic
+        AGE_REQ_ADULT, // Goron tunic
+        AGE_REQ_ADULT  // Zora tunic
+    },
+    {
+        AGE_REQ_NONE,  // Scale
+        AGE_REQ_NONE,  // Kokiri boots
+        AGE_REQ_ADULT, // Iron Boots
+        AGE_REQ_ADULT  // Hover Boots
+    },
+};
+
+// Controls if the icon is grayed out or not depending on age
+u8 gItemIconAgeInfo[] = {
```

Comment:
```suggestion
// Controls if the icon in the subscreen is grayed out or not depending on age (indexed by `ItemID`)
u8 gItemIconAgeReqs[] = {
```
(is it indexed by ItemID?)
(I assume in the subscreen/pause menu?)

Reply:
So I checked and it is indeed indexed by ItemID:
https://github.com/zeldaret/oot/blob/c2195e23fe2d4431a02dda7a9426194bdb036832/src/overlays/actors/ovl_kaleido_scope/z_kaleido_scope_PAL.c#L2666-L2668
and assuming texture symbol names in `gItemIcons` are accurate it lines up with ItemID